proposals/dictionary-expressions.md

```diff
@@ -220,33 +281,48 @@ The new rules above represent a breaking change: For types that are a valid conv
 > For the create method:
 >   - The method must have a single parameter of type System.ReadOnlySpan<E>, passed by value, and there is an identity conversion from E to the iteration type of the collection type.
 >    - ```diff
->      + Or, the method have a single parameter of
+>      + Or, the method has a single parameter of
 >      + `IEnumerable<KeyValuePair<TKey, TValue>>` and iteration type
 >      + of the collection type is the same `KeyValuePair<,>` type. 
+>   - The method has two parameters, where the first is an `IEqualityComparer<TKey>` and the second follows the rules of the 'single parameter' rule above.  This method will be called if the collection expression includes an initial `comparer: expression`.
 > ```
 
 This would allow `ImmutableDictionary<TKey, TValue>` to be annotated with `[CollectionBuilder(typeof(ImmutableDictionary), "CreateRange")]` to light up support for creation.
 
-### Open question 1
+### Question: Efficient 'Create' methods
 
 We could consider not adding this rule, and instead still require the create method to take a `ReadOnlySpan<E>`.  However, that would require the BCL to then add such a method to `ImmutableDictionary` as the existing `CreateRange` methods take an `IEnumerable`.
 
-### Open question 2
+Conclusion: TBD.  Working group recommendation.  We will defer to the BCL here.  They will have to add `[CollectionBuilder]` annotations to light up anyways.  They can either point these at the existing `CreateRange` methods, or they could point at new methods that take spans.
+
+Pointing at the existing method would have the positive that it would add to the BCL surface area.  But it would have the negative of forcing going through interfaces to create an immutable dictionary.
```

Comment:
Consider merging these two paragraphs and simplifying:
```suggestion
Pointing at the existing method would force going through interfaces to create an immutable dictionary.
Pointing at a new method would allow for more efficient, stack-based, initialization, but would increase the BCL surface area.
```